src/main/scala/info/bethard/anafora/Anafora.scala

```diff
@@ -6,12 +6,18 @@ import com.codecommit.antixml.XML
 
 
 object Data {
-  def fromPaths(xmlPath: String, textPath: String) = apply(
-    XML.fromSource(io.Source.fromFile(xmlPath)),
-    io.Source.fromFile(textPath).mkString)
-  def apply(xml: Elem, text: String) = new Data(xml, text)
+  def fromPaths(xmlPath: String, textPath: Option[String]) =textPath match {
```

Comment:
I believe this could be more concisely written as:
```
apply(XML.fromSource(io.Source.fromFile(xmlPath)), textPath.map(p => io.Source.fromFile(p).mkString)
```